experimental/generation/generator/package.json

```diff
@@ -5,15 +5,17 @@
   "author": "Microsoft",
   "bugs": "https://github.com/Microsoft/BotBuilder-Samples/issues",
   "dependencies": {
-    "@microsoft/bf-cli-command": "^4.8.0",
+    "@microsoft/bf-cli-command": "^4.9.0-RC6",
+    "@microsoft/bf-lu": "^4.10.0-preview.130225",
```

Comment:
>"^4.10.0-preview.130225", [](start = 24, length = 25)

this doesn't seem right.

Reply:
That is correct.  I have to pick up bug fixes.

---
In reply to: [423897684](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423897684) [](ancestors = 423897684)